config/201-clusterrolebinding.yaml

```diff
@@ -26,3 +26,22 @@ roleRef:
   kind: ClusterRole
   name: binding-system-admin
   apiGroup: rbac.authorization.k8s.io
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: sink-binding-source-observer
+  labels:
+    duck.knative.dev/source: "true"
```

Comment:
This is the wrong place for this, and I think I already made this change in 200-clusterole.yaml